admin/translations/en.json

```
@@ -94,6 +94,12 @@
         "title": "Mass modification of organisations",
         "description": "Note: Only columns with a value will be modified.",
         "notifications": {
+          "error": {
```

Dans les fichiers de traductions on trouve à la fois `"errors": {` et `"error": {`, mais la version au pluriel semble plus logique et c'est aussi celle qui est en plus grand nombre, d'où la suggestion suivante : 

```suggestion
          "errors": {
```